package.json

```diff
@@ -29,6 +29,7 @@
   "main": "scripts/index.js",
   "dependencies": {
     "angular-history": "git+https://github.com/decipherinc/angular-history.git#v0.8.0",
+    "superdesk-planning": "git+https://github.com/superdesk/superdesk-planning.git#1f04215",
```

Comment:
I wonder if this should be here. If yes, then why not include it straight into client-core if it is already permanently part of it? Do we really need a separate node module given that it is not optional?

It seems like this kind of defeats the whole purpose of having external modules... What do the others thinks?